Shipr-Helm/templates/shipr-frontend-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.frontend.name }}
+  labels:
+    app: {{ .Values.frontend.name }}
```

Comment:
Again, why is there only one hardcoded label

Reply:
Fixed